tests/chat/models/test_async_openai_chat.py

```diff
@@ -0,0 +1,129 @@
+"""Tests for mirascope async openai chat API model classes."""
```

Comment:
you're missing some tests around `Prompt` vs. `messages` and `str` prompts etc.

I have those tests for the openai_chat so reference that